fastlane/Fastfile

```diff
@@ -235,7 +235,7 @@ import "./ScreenshotFastfile"
     ios_finalize_prechecks(options)
     unless ios_current_branch_is_hotfix
       ios_update_metadata(options)
-      ios_lint_localizations(input_dir: 'WordPress/Resources', allow_retry: true)
+      #ios_lint_localizations(input_dir: 'WordPress/Resources', allow_retry: true)
```

Comment:
From e7a2760a7ad2162e31d87d0bfb1536f5154db94a, it's clear that this was not an accidental leftover.

What's the plan to fix it? I'm fine to merge with it if it's a "long" plan.

Reply:
We need to break a string into multiple in GlotPress, but then we need to wait for it to be translated, so can't be done within the scope of this PR :(

Reply:
Gotcha 👍